### ISSUE TITLE
RC4 Cipher

### ISSUE THREAD

**Issue by shrinidhihegde3** [shrinidhihegde3/Snippify-Hub#60]
A stream cipher where each bit or byte is encrypted separately. RC4 was widely used but is now considered insecure due to vulnerabilities.
Add input and output field so that people can experiment

**Comment by SanskariWolf** [comment]
Can you please assign this issue to me? I would like to work over it.

**Comment by shrinidhihegde3** [comment]
Contribute inside Ciphers folder

**Comment by shrinidhihegde3** [comment]
any updates?

**Comment by SanskariWolf** [comment]
Yup, i am nearly done with the work, will be creating the merge request by 17th of October.
